<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllOnlineSupportUsers</class>
 <widget class="QDialog" name="GetAllOnlineSupportUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Online Support Users</string>
   </property>
   <widget class="QListWidget" name="listOnSupportComm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>341</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>User Details</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>320</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection ID</string>
    </property>
    <widget class="QLabel" name="lblOnSupportCntnIdComm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>320</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>User Type</string>
    </property>
    <widget class="QLabel" name="lblOnSupportUsrTypeComm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>320</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Name</string>
    </property>
    <widget class="QLabel" name="lblOnSupportNameComm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>320</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Phone Number</string>
    </property>
    <widget class="QLabel" name="lblOnSupportPhComm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>320</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Address</string>
    </property>
    <widget class="QLabel" name="lblOnSupportAddressComm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOnSupportClose">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
